CHANGELOG.rst

```diff
@@ -2,6 +2,29 @@
 Release Notes
 =============
 
+.. _Release Notes_7.21.1:
+
+7.21.1
+======
+
+.. _Release Notes_7.21.1_Prelude:
+
+Prelude
+-------
+
+Release on: 2020-07-21
+
+- Please refer to the `7.21.1 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7211>`_ for the list of changes on the Core Checks
```

Comment:
I don't think there are any integrations changes (I might be wrong), so let's remove this